test/functional/job/job_spec.lua

```diff
@@ -228,10 +228,10 @@ describe('jobs', function()
     it('returns a list of status codes', function()
       source([[
       call rpcnotify(g:channel, 'wait', jobwait([
-      \  jobstart([&sh, '-c', 'sleep 0.10; exit 4']),
-      \  jobstart([&sh, '-c', 'sleep 0.110; exit 5']),
-      \  jobstart([&sh, '-c', 'sleep 0.210; exit 6']),
-      \  jobstart([&sh, '-c', 'sleep 0.310; exit 7'])
+      \  jobstart([&sh, &shellcmdflag, 'sleep 0.10; exit 4']),
+      \  jobstart([&sh, &shellcmdflag, 'sleep 0.110; exit 5']),
+      \  jobstart([&sh, &shellcmdflag, 'sleep 0.210; exit 6']),
+      \  jobstart([&sh, &shellcmdflag, 'sleep 0.310; exit 7'])
```

Comment:
Actually you should have here `jobstart(split(&shell) + split(&shellcmdflag) + ['sleep 0.10; exit 4'])`. Documentation that says that `jobstart('{cmd}')` has something to do with

    			:call jobstart([&shell, &shellcmdflag, '{cmd}'])

is incorrect. My variant is neither because &shell and &shellcmdflag are split with quotes taken into account (which I find rather useless because I have not found where it is unquoted and additionally it does not take backslash into account), but this is closer.